go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/ethereum/go-ethereum
 
-go 1.21
-
-toolchain go1.21.6
+go 1.22
```

Comment:
@0x00101010 see this comment from @ajsutton regarding Go 1.22: https://github.com/ethereum-optimism/op-geth/pull/389#issuecomment-2375606462

You can switch back to 1.21 by downgrading the go-verkle library as done in https://github.com/ethereum-optimism/op-geth/pull/389

Reply:
oh nice 🙌 resolving